src/standard/expr.mli

```diff
@@ -853,6 +854,648 @@ module Term : sig
     include Dolmen_intf.Term.Tptp_Thf_Core_Const with type t := t
     (** Satisfy the required interface for the typing of tptp's Thf. *)
 
+    val eqs : int -> ty id
```

Comment:
I think it'd be better here to use either `t` or `term_const` (`ty id` is rather obscure and it's easy to forgot that it is a term constant and not a type constant), same for all the rest of the file

Reply:
Yes I agree, it's done.